Lib/unittest/case.py

```diff
@@ -13,15 +13,20 @@
 import types
 
 from . import result
-from .util import (strclass, safe_repr, _count_diff_all_purpose,
-                   _count_diff_hashable, _common_shorten_repr)
+from .util import (
+    strclass,
+    safe_repr,
+    _count_diff_all_purpose,
+    _count_diff_hashable,
+    _common_shorten_repr,
+)
```

Comment:
Your pr has a lot of formatting changes.  I suggest you to revert all this stuff, include only changes, relevant to your issue.

Reply:
It looks like the whole file was formatted with Black. As a maintainer of Black I approve, but as a maintainer of CPython I prefer that PRs are more focused.